src/components/Username.tsx

```diff
@@ -0,0 +1,20 @@
+import { classes as commentClasses } from '../style/comments.st.css';
+import { classes } from '../style/main.st.css';
+
+interface IUsername {
+    name: string;
+    header: boolean;
+}
+const Username = ({ name, header }: IUsername) => (
+    <>
+        {header ? (
+            <span className={commentClasses.commentsHeader}>
+                <span className={classes.boldText}>{name} &#8226; Following</span>
```

Comment:
"&#8226" what is this escape character?

Reply:
the dot on the header between the userName and "Following"